pallets/gear/src/tests.rs

```diff
@@ -549,21 +549,35 @@ fn delayed_send_user_message_payment() {
             }
         ));
 
-        // Mailbox should be empty
+        // Mailbox should be empty.
         assert!(MailboxOf::<Test>::is_empty(&USER_2));
 
         run_to_next_block(None);
 
+        // Check that last event is UserMessageSent.
+        let last_event = match System::events().last().map(|r| r.event.clone()) {
```

Comment:
We already had fn for last event, if Im not mistaken
+ it's not unreachable, it's just panic cases (it doesn't change anything in fact, but its better conceptually and improves searching all over the project of unreachable places)

Reply:
same in other tests

Reply:
done